spec/models/plugins/shared_examples.rb

```diff
@@ -37,6 +37,13 @@
     expect( subject.liquid_data({form_values: {email: 'neal@test.com'}})[:outstanding_fields]).to eq []
   end
 
+  it 'deletes the form when it is deleted' do
+    subject.form = form
+    subject.save!
+    expect{ subject.destroy! }.not_to raise_error
```

Comment:
Yes! glad you asked. We have an outstanding issue where we cannot delete plugins created before your formable migration. Deleting the plugin tries to cascade the delete to the associated form (because of `dependent: destroy`) but that fails because of the foreign key constraint (the form is still referenced by the plugin row that has not yet been deleted). 

I tried to reproduce with this test, but soon discovered that I can't reproduce with new records - only existing records. I assume it was something about the migration behaving weirdly? Do you have any insights?